web/styles/dark_theme.css

```diff
@@ -350,7 +350,8 @@
     :disabled,
     input:not([type="radio"]):read-only,
     textarea:read-only,
-    #organization-permissions .dropdown-widget-button:disabled {
+    #organization-permissions .dropdown-widget-button:disabled,
+    #stream-advanced-configurations .dropdown-widget-button:disabled {
         color: inherit;
         opacity: 0.5;
     }
```

Comment:
Looking at this, I'm very much left wondering if we can just make these the styles for `.dropdown-widget-button:disabled` , perhaps using CSS variables, rather than having longer lists of selectors here. But I guess we can do it as a follow-up.